components/templates/templates.js

```diff
@@ -330,5 +337,27 @@ if (!utils.templates) {
         target.appendChild(templates.item(c));
       }
     };
+
+    if (document.registerElement) {
```

Comment:
Maybe we could move this to an internal function called <init> and we could execute this here, at the end of the module. <init> or what name you want. Therefore anybody who wants to add something could find where in order not to have code scattered. IMHO this is how I see this code although it is just my taste and I don't have a strong opinion.